python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -958,6 +959,160 @@ def corr(self, method="pearson", min_periods=1):
 
         return res
 
+    def cov(self, min_periods=0, ddof=1):
+        """
+        Compute the pairwise covariance among the columns of a DataFrame,
+        excluding NA/null values.
+
+        The returned data frame is the covariance matrix of the columns of
+        the DataFrame.
+
+        Both NA and null values are automatically excluded from the
+        calculation.(See the note below about bias from missing values)
+        A threshold can be set for the minimum number of observations
+        for each value created.Comparisons with observations below this
+        threshold will be returned as NaN.
+
+        This method is generally used for the analysis of time series data to
+        understand the relationship between different measures across time.
+
+        Parameters
+        ----------
+        min_periods: int, optional
+            Minimum number of observations required per pair of columns
+            to have a valid result.
+
+        ddof: int, optional
+            Delta degrees of freedom, default is 1.
+
+        Returns
+        -------
+        DataFrame
+            Covariance matrix.
+
+        Notes
+        -----
+        Returns the covariance matrix of the DataFrame’s time series.
+        The covariance is normalized by N-ddof.
+
+        For DataFrames that have Series that are missing data
+        (assuming that data is missing at random) the returned covariance
+        matrix will be an unbiased estimate of the variance and covariance
+        between the member Series.
+
+        However, for many applications this estimate may not be acceptable
+        because the estimate covariance matrix is not guaranteed to be
+        positive semi-definite. This could lead to estimate correlations
+        having absolute values which are greater than one, and/or a
+        non-invertible covariance matrix. See Estimation of covariance
+        matrices for more details.
```

Comment:
Link this to Wikipedia, as the original pandas docstring does.
```suggestion
        non-invertible covariance matrix. See `Estimation of covariance
        matrices <https://en.wikipedia.org/wiki/Estimation_of_covariance_matrices>`_
        for more details.
```
You might need to add `# noqa: E501` after the end of the docstring `"""` if the URL is too long for flake8. I'll add a suggestion there too.